### PR TITLE
gumsare perfection

### PR DESCRIPTION
**Pull request by singhRaw**
perfection is everything

### CODE REVIEW

File: l1.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 print('Hello World')
 print('Evyerthing is beautiful')
+print('gums are perfection')
```
Comment (RawendraSinghKushwaha) [inline l1.py:4]: this is cool @singhRaw

Comment (singhRaw) [reply l1.py:4]: i know right, its so cool

Comment (RawendraSinghKushwaha) [reply l1.py:4]: thumbs up sire

**Review by RawendraSinghKushwaha — APPROVED** [review]
this works

**Pull request merged**